### ISSUE TITLE
[TW-1820] Install with -DLANGUAGE=2 flag not work.

### ISSUE THREAD

**Issue by taskwarrior** [GothenburgBitFactory/taskwarrior#1836]
_E. Manuel Cerr'on Angeles on 2016-06-14T16:47:54Z says:_ 

 cmake -DLANGUAGE=2 -DCMAKE_BUILD_TYPE=release .

CMAKE_SYSTEM_NAME Linux
* - Looking for SHA1 references
* - Looking for GnuTLS
* - Looking for libuuid
* - Found libuuid
* - Configuring cmake.h
* - Configuring man pages
* - Configuring done
* - Generating done
* - Build files have been written to: /home/xerron/Vídeos/temp/task-2.5.1

make

[ 52%] Building CXX object src/CMakeFiles/task.dir/util.cpp.o
/home/xerron/Vídeos/temp/task-2.5.1/src/util.cpp: In function ‘void signal_handler(int)’:
/home/xerron/Vídeos/temp/task-2.5.1/src/util.cpp:67:28: error: ‘STRING_ERROR_CONFIRM_SIGINT’ was not declared in this scope
     std::cout << "\n\n" << STRING_ERROR_CONFIRM_SIGINT << "\n";
                            ^
src/CMakeFiles/task.dir/build.make:830: fallo en las instrucciones para el objetivo 'src/CMakeFiles/task.dir/util.cpp.o'
make[2]: *** [src/CMakeFiles/task.dir/util.cpp.o] Error 1
CMakeFiles/Makefile2:208: fallo en las instrucciones para el objetivo 'src/CMakeFiles/task.dir/all'
make[1]: *** [src/CMakeFiles/task.dir/all] Error 2
Makefile:149: fallo en las instrucciones para el objetivo 'all'
make: *** [all] Error 2

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2016-06-14T16:47:54Z
Modified: 2016-06-14T23:45:30Z
```

**Comment by taskwarrior** [comment]
_Paul Beckingham on 2016-06-14T23:42:08Z says:_ 

 Localized files with missing string added.

**Comment by taskwarrior** [comment]
_Paul Beckingham on 2016-06-14T23:45:30Z says:_ 

 Thank you for reporting this, and sorry for the inconvenience.

I have addressed this in the 2.6.0 codebase, but I cannot change the 2.5.1 release.  I can, however provide you with a patched esp-ESP.h file (attached to this issue), which you can use as a drop-in replacement for the bad file you have.  This will allow you to complete the build, but the missing string is not localized - it is in English, which is all I can provide.

**Comment by pbeckingham** [comment]
[TW-1820_esp-ESP.h.txt](https://github.com/GothenburgBitFactory/taskwarrior/files/1756437/TW-1820_esp-ESP.h.txt)
